docs/devel/making-release-notes.md

```diff
@@ -2,17 +2,21 @@
 This documents the process for making release notes for a release.
 
 ### 1) Note the PR number of the previous release
-Find the PR that was merged with the previous release.  Remember this number
+Find the most-recent PR that was merged with the previous .0 release.  Remember this as $LASTPR.
 _TODO_: Figure out a way to record this somewhere to save the next release engineer time.
 
+Find the most-recent PR that was merged with the current .0 release.  Remeber this as $CURRENTPR.
+
 ### 2) Build the release-notes tool
```

Comment:
`build/make-release-notes.sh` takes care of building so this step should be called either "Run" or "Build/Run".